README.md

```diff
@@ -9,11 +10,27 @@ In the case of the onprem provider, `dcos-launch` provides the capability to cre
 
 The primary purpose of `dcos-launch` is to provide a turn-key deployment experience across a wide variety of configurations without having to handle any of the lower-level cloud APIs.
 
-## Installation
+# Usage
+
+`dcos-launch` is backwards compatible to older verions of DC/OS. Thus, it can be used as a development or CI tool for deploying clusters into your workflow.
+
+## Requirements
+* Linux operating system
+* SSH client installed on localhost
+
+### Credentials
```

Comment:
I would replace the contents of this section with the "setup your credentials" section of this wiki page: https://wiki.mesosphere.com/pages/viewpage.action?pageId=117084126

Reply:
addressed